src/test_serializer.rs

```diff
@@ -89,6 +90,24 @@ fn serialize_item_byteseq_with_param() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+#[test]
+fn serialize_validated_item_byteseq_with_param() -> Result<(), Box<dyn Error>> {
```

Comment:
This is currently just copying a single test to check whether the idea works out as anticipated. If the approach is approved, we can add more test coverage.

Reply:
Removed as no more `Validated*` enum variants exist.